lib/twine/placeholders.rb

```diff
@@ -51,5 +55,20 @@ def convert_placeholders_from_android_to_twine(input)
       # %s -> %@
       input.gsub(placeholder_regex, '\1@')
     end
+
+    # http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/mx/resources/IResourceManager.html#getString()
+    # http://soenkerohde.com/2008/07/flex-localization/comment-page-1/
+    def convert_placeholders_from_twine_to_flash(input)
```

Comment:
I initially separated the placeholder conversion logic from the formatters, because some formatters share their conversion logic (for example Tizen and Android). Even if not the complete conversion process is shared, at least some aspects (like the "match a Twine printf-style placeholder") are shared across formatters. My reasoning was, that it's better to keep these aspects together than scattered over multiple classes which makes it a lot harder to keep track where a fix needs to be applied.